hoot-core-test/src/test/cpp/hoot/core/algorithms/MaximalNearestSublineTest.cpp

```diff
@@ -63,55 +63,60 @@ using namespace std;
 
 class MaximalNearestSublineTest : public CppUnit::TestFixture
 {
-    CPPUNIT_TEST_SUITE(MaximalNearestSublineTest);
-    CPPUNIT_TEST(runTest);
-    CPPUNIT_TEST(funnyCurveTest);
-    CPPUNIT_TEST(oneShortTest);
-    CPPUNIT_TEST_SUITE_END();
+  CPPUNIT_TEST_SUITE(MaximalNearestSublineTest);
+  CPPUNIT_TEST(runTest);
+  CPPUNIT_TEST(funnyCurveTest);
+  CPPUNIT_TEST(oneShortTest);
+  CPPUNIT_TEST_SUITE_END();
```

Comment:
Some changes here are my OCD making things all the same across tests, like these extra spaces.